src/drive/locales/en.json

```diff
@@ -513,5 +513,15 @@
       "disconnect": "Refuse and disconnect",
       "error": "Something went wrong, please try again later"
     }
+  },
+  "manifest": {
+    "permissions": {
+      "contacts": {
+        "description": "Required to share files with your contacts"
+      },
+      "groups": {
+        "description": "Required to share files with your groups"
+      }
+    }
```

Comment:
Do you know the action you'll have to do after merging this PR (concerning tx)? 

If not : after merged : tx push -S